sdk/tables/Azure.Data.Tables/src/Queryable/TypeSystem.cs

```diff
@@ -0,0 +1,151 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Reflection;
+
+namespace Azure.Data.Tables.Queryable
+{
+    internal static class TypeSystem
```

Comment:
Are most of these things generaly applicable to OData queries?

Reply:
No, I think these may be unnecessary. Code coverage data indicates that at least given the current suite of tests, the code path that touches these is never hit.